package.json

```diff
@@ -34,8 +34,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration": "npm run stop && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) && npm run stop",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*",
+    "test:integration": "node src/integration/utils/runTests/index.js",
+    "test:integration:ci": "JEST_SILENT_REPORTER_DOTS=true npm run test:integration -- --ci --reporters=jest-silent-reporter",
```

Comment:
I can't seem to run `npm test` on my local machine without snapshot errors – I get the following hint:
```
This is likely because this test is run in a continuous integration (CI) environment in which snapshots are not written by default.
```
Also, is there a reason why `scripts['test']` (as opposed to `scripts['test:ci']`) runs `test:integration:ci`?

Reply:
@simonsinclair I haven't generated the snapshots because this would add roughly 600 more files to the PR. I'll generate the snapshots as my last commit and ask for a quick once over again before merging.